src/Sylius/Bundle/WebBundle/Resources/views/Backend/Promotion/create.html.twig

```diff
@@ -2,6 +2,15 @@
 
 {% from 'SyliusResourceBundle:Macros:actions.html.twig' import create %}
 
+{% block javascripts %}
+    {{ parent() }}
+    {% javascripts output='assets/compiled/backend_promotions.js'
+    'bundles/syliuspromotion/js/sylius-promotion.js'
+    %}
+    <script type="text/javascript" src="{{ asset_url }}"></script>
```

Comment:
It's already a quick fix, we'll change whole CRUD pages in weeks, so I think there is no need to change it ;)